src/style.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+  color: #3e4e61;
+}
```

Comment:
Set default font-weight that you want to use for all elements and add font styles for placeholder